src/index.html

```diff
@@ -15,8 +15,107 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="./reset.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./fons.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
The Roboto font is being loaded both locally from `fons.css` and via Google Fonts. This redundancy can be avoided by choosing one method to load the font to optimize performance.